- `nixStatic` is broken due to a pre-existing failure in a dependency
- `nix-eval-jobs` is new:
```
FAILED: src/nix-eval-jobs.p/nix-eval-jobs.cc.o 
g++ -Isrc/nix-eval-jobs.p -Isrc -I../src -I/nix/store/w639nhbzmil3g061if2rnjhbfngqw6rf-nix-2.5.1-dev/include/nix -I/nix/store/ql99k1yggw3gkpka7bp6zqvkn8fsv6xx-boehm-gc-8.0.6-de>
../src/nix-eval-jobs.cc:32:8: warning: base class 'struct nix::MixEvalArgs' has accessible non-virtual destructor [-Wnon-virtual-dtor]
   32 | struct MyArgs : MixEvalArgs, MixCommonArgs
      |        ^~~~~~
../src/nix-eval-jobs.cc:32:8: warning: base class 'class nix::MixCommonArgs' has accessible non-virtual destructor [-Wnon-virtual-dtor]
../src/nix-eval-jobs.cc:32:8: warning: 'struct MyArgs' has virtual functions and accessible non-virtual destructor [-Wnon-virtual-dtor]
../src/nix-eval-jobs.cc: In function 'void worker(nix::EvalState&, nix::Bindings&, nix::AutoCloseFD&, nix::AutoCloseFD&, const Path&)':
../src/nix-eval-jobs.cc:191:72: error: no matching function for call to 'printValueAsJSON(nix::EvalState&, bool, nix::Value&, std::stringstream&, nix::PathSet&)'
  191 |                   printValueAsJSON(state, true, *metaValue, ss, context);
      |                                                                        ^
In file included from ../src/nix-eval-jobs.cc:20:
/nix/store/w639nhbzmil3g061if2rnjhbfngqw6rf-nix-2.5.1-dev/include/nix/value-to-json.hh:13:6: note: candidate: 'void nix::printValueAsJSON(nix::EvalState&, bool, nix::Value&, co>
   13 | void printValueAsJSON(EvalState & state, bool strict,
      |      ^~~~~~~~~~~~~~~~
/nix/store/w639nhbzmil3g061if2rnjhbfngqw6rf-nix-2.5.1-dev/include/nix/value-to-json.hh:13:6: note:   candidate expects 6 arguments, 5 provided
/nix/store/w639nhbzmil3g061if2rnjhbfngqw6rf-nix-2.5.1-dev/include/nix/value-to-json.hh:16:6: note: candidate: 'void nix::printValueAsJSON(nix::EvalState&, bool, nix::Value&, co>
   16 | void printValueAsJSON(EvalState & state, bool strict,
      |      ^~~~~~~~~~~~~~~~
/nix/store/w639nhbzmil3g061if2rnjhbfngqw6rf-nix-2.5.1-dev/include/nix/value-to-json.hh:16:6: note:   candidate expects 6 arguments, 5 provided
ninja: build stopped: subcommand failed.
```

--- a/pkgs/tools/package-management/nix-eval-jobs/default.nix
+++ b/pkgs/tools/package-management/nix-eval-jobs/default.nix
@@ -11,12 +11,12 @@
 }:
 stdenv.mkDerivation rec {
   pname = "nix-eval-jobs";
-  version = "0.0.1";
+  version = "0.0.2";
   src = fetchFromGitHub {
     owner = "nix-community";
     repo = pname;
     rev = "v${version}";
-    hash = "sha256-LTMW4356f8pvIyfYdOyZbF9yzU8MH9mryQgB4LrwZMI=";
+    hash = "sha256-kZw/nPJqWuBMMnDWsWF3oMY93QYVRem1XTbaxdbQ2oM=";
   };
   buildInputs = [
     boost

